apps/dashboard-for-dapps/src/routes/index.tsx

```diff
@@ -58,14 +49,6 @@ export const Route = createFileRoute("/")({
   },
 });
 
-function transformBase85Image(src: string) {
-  const prefix = "data:image/jpeg;base85,";
-
-  return `data:image/png;base64,${Base64Codec.encode(
-    ascii85.decode(src.substring(prefix.length)),
-  )}`;
-}
-
```

Comment:
How does image rendering work without this? I remember picking base85 because of superior compression, but it's a format browsers don't render it natively, which is why we're converting it with this function or similar. I may be misunderstanding, but it seems that now you're calling `openImageInNewTab` with base85 content. Is this a format browsers render now, or am I missing something else?

Reply:
we used to use this method indeed. but appearantly it was refactored to openInNewTab. didn't notice that it doesn't work properly but taking a look rn